README.md

```diff
@@ -1,11 +1,16 @@
-# glusterapi-python
+# python-gluster-mgmt-client
 
 Python bindings for gluster management APIs
```

Comment:
I think the introduction should note that this is specifically  target at GlusterD2 (the "up & coming" management service for gluster).

Reply:
done.